docs/dev/stream/state/state.md

```diff
@@ -533,8 +533,8 @@ in that:
 
 ## Using Operator State
 
-To use operator state, a stateful function can implement either the more general `CheckpointedFunction`
-interface, or the `ListCheckpointed<T extends Serializable>` interface.
+To use operator state, a stateful function can implement the `CheckpointedFunction`
```

Comment:
Just thinking out loud, not for this pr, but we are probably going to need to u[date the data stream docs to avoid the phrase "stateful function"

Reply:
😂